LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2009-2015 Spree Commerce and contributors.
```

Comment:
Because this was started as a fork of the original extension, I don't think we should be removing Spree's name from the license. Maybe we can just keep both copyright notices.

Reply:
I made an amend with only that line changed, just to speed up the changes:
https://github.com/solidusio-contrib/solidus_volume_pricing/commit/a6fb9de05ce09b9941077467e2d54e5dc118cb82#diff-37854d19817c792316d481f5beb93cc7R1